roles/oVirt.hosts/tasks/main.yml

```diff
@@ -35,6 +35,7 @@
     override_iptables: true
     timeout: "{{ item.timeout | default(ovirt_hosts_add_timeout) }}"
     poll_interval: "{{ item.poll_interval | default(20) }}"
+    hosted_engine: "{{ item.hosted_engine | default(omit) }}"
```

Comment:
Could you please also add a description of this option into [README.md](https://github.com/oVirt/ovirt-ansible-infra/blob/master/README.md#hosts) ?

Reply:
I just want to ack this, I've been super busy, I will get this done..